### ISSUE TITLE
Bring Flare up to speed with Jade

### ISSUE THREAD

**Issue by luchia** [TeamMentor/TM_4_0_Design#522]
There have been a lot of design changes to Jade in the last couple of months that I haven't had time to migrate over to Flare as well, due to time constraints. This is going to be a large issue with a couple of weeks work in it. It needs to be done before Flare is launched.

I will keep updating this checklist as and when I find more things.

### Articles Directory
- [x] Article titles need to be changed to match Jade
- [x] Add margin-right to icon in tabs-header 
- [x] Replace save icon with trash icon on my-articles template

### Getting Started Directory
- [x] Make buttons only have half-width (too much button)
- [x] Fix show button on sign-up forms
- [x] Less padding on forms (narrow-width for non-logged in pages makes it *very* narrow)
- [x] Build pwd reset pages
- [x] Align icons in #summary

### Help Directory
- [x] Fix margin on help navigation

### Home Directory
- [x] Fix .postfix buttons on search bar from having too wide a width
- [x] Fix buttons on save/share divs (currently can't be seen)
- [x] Fix margin on filters div
- [x] Find a better solution for main-app-view mixin
- [x] Navigate should have the search bar still

### Landing Pages Directory
- [x] Smaller button in call-to-action div
- [x] Fix spacing on features page
- [x] Fix landing pages to match new navigation built in #522

### New User Onboard Directory
- [x] Update copy
- [x] Update styling to stand out from the regular app (at the moment, text looks like it's an article in the app)
- [x] Migrate the templates that are built in HTML to [Foundations Joyride](http://foundation.zurb.com/docs/components/joyride.html), a built in feature specifically for onboarding
- [x] Smaller buttons

### User Directory
- [x] Fix width on TOC page
- [x] Add trash icon to 'Danger Zone' delete account button

### CSS
- [x] Add margin to list-view-article buttons so they sit lower in the panel
- [x] Replace ids with classes (where applicable [nb: rookie error])
- [x] Less padding between everything in general (wasting limited space)
- [x] Update typography colours
- [x] Remove font-weight on h4s
- [x] Remove application margin/padding bottom
- [x] Fix hover not covering whole item (lists)

### General 
- [x] Add SI logo to copyright mixin
- [x] Figure out a solution to make the search bar stand out more than the rest of the elements in the application *(Edit: See issue #576 for solution)*

**Comment by luchia** [comment]
Prioritising my work and making this a P0 so I have more levels within my Sprint 3 issues.

**Comment by luchia** [comment]
I'm almost finished this issue and will be able to pass it on to be closed after tomorrow.

**Comment by luchia** [comment]
This issue is ready for QA/closing. I don't know if anyone wants to "check" this because it was more things that I knew I needed to do to make Flare look like Jade does - a lot of the details are very small and maybe not checkable. I don't know, assigning to @roman87 so he can QA and/or close.

**Comment by DinisCruz** [comment]
merged into dev

**Comment by romichg** [comment]
Done.
